AirportDepartures.playground/Contents.swift

```diff
@@ -16,9 +16,51 @@ import UIKit
 //: e. Use a `String?` for the Terminal, since it may not be set yet (i.e.: waiting to arrive on time)
 //:
 //: f. Use a class to represent a `DepartureBoard` with a list of departure flights, and the current airport
-
-
-
+enum FlightStatus: String {
+    case enRoute
+    case scheduled
+    case canceled
+    case delayed
+    case landed
+    case scheduledOnTime
+    case scheduledLate
+}
+
+struct Airport {
+    
+    var airportDestination: String
+    var airportArrival: String
+}
+
+struct Flight {
+    
+    var departureTime: Date?
+    var terminal: String?
+    var destination: String
+    var flight: String
+    var status: String
+    
+}
+
+class DepartureBoard {
+    
+    var flights:[Flight]
+    
+    init(flights: [Flight]) {
+        self.flights = flights
+        
+    }
+}
+//let y = Flight(departureTime: Date(), terminal: "2", destination: "NY", flight: "225", status: "Canceled")
+//let x = DepartureBoard(flights: [y])
+
+
+
+//init(name: String, address: String, coffees: [Coffee]) {
+//       self.name = name
+//       self.address = address
+//       self.coffees = coffees
+//   }
 //: ## 2. Create 3 flights and add them to a departure board
```

Comment:
Remember to read the instructions thoroughly.  This line is asking you to create flights and add them to a departure board.  (In this instance you have not yet made a departure board, so that will need to be created)